project-setup/base-documentation.md

```diff
@@ -0,0 +1,113 @@
+creating this documentation should be to make getting started as much a self
+service process as possible with frequently asked questions answered in standard
+documentation format.
+
+If it does not yet exist, create a **README.md** for your project. It should
+contain:
+
+* The [mission of the
+  project](https://producingoss.com/en/producingoss.html#mission-statement) in
+  as a concise format as possible. It should answer what the project's purpose
+  is and enable contributors to make a good first guess whether a suggested
+  feature will likely be in scope for the project, or not.
+* A "Getting Started" section for downstream users of the project. It should
+  explain how to setup/ integrate the project artifacts as well as an
+  explanation of some of the first typical steps for first time users.
+* Deeper documentation for project users - or a link to that.
+* Documentation needed for making modifications to the project - or a link to
+  that.
+* Documentation on how to contribute to the project - or a link to that.
+* A "Getting involved" section that explains which public, archived, linkable
+  communication channels the project uses. This should include a link to the
+  project issue tracker, but also to any further discussion media used.
+* A "Who we are" section explaining who the Trusted Committers behind the
+  project are - with an explanation that instead of contacting these people
+  privately the public communication channels above should be used for
+  communication.
+* An explanation of what the criteria are for the project to turn contributors
+  into Trusted Committers - if that path exists.
+
+If the explanation of the steps to make a contribution are too involved, create
+a separate **CONTRIBUTING.md** document. This document should answer frequently
+asked questions that contributors have asked in the past. There is no need to
+provide a comprehensive book up front. Rather share information that has proven
+needed by contributors. Likely it will touch upon one or more of the following
+topics:
+
+* How to checkout the project source code from version control.
+* How to make modifications to the project (potentially including information on
+  coding guidelines.)
+* How to build the project.
+* How to run tests to make sure the above modifications aren't introducing new
+  bugs.
+* How to submit your modifications back to the project.
+* Some information on which turnaround time to expect for modifications made.
+
+
+There are literally tons of good examples for how to write a README.md and what kind
+of information to include in a CONTRIBUTING.md file in various open source projects. 
+Pages like [how to write a readme that rocks](https://m.dotdev.co/how-to-write-a-readme-that-rocks-bc29f279611a), 
+[Open Source Guide from GitHub](https://opensource.guide/) as well as
+the book [Producing Open Source](https://producingoss.com/en/producingoss.html) all
```

Comment:
I don't actually see good information on a README at the Producing OSS link. Am I just missing it?

Reply:
There is no chapter on writing a good README. The book doesn't say which form to choose to provide the documentation that is often (but not always) found in or linked to in a README.

However in the Getting Started Chapter (https://producingoss.com/en/producingoss.html#starting-from-what-you-have) it does provide a fairly extensive list of things that fellow host team members, users and contributors will need. I don't expect an InnerSource project to cover all of those aspects in their documentation - I find the list helpful as inspiration for what one could cover.